dev/intellij-setup.md

```diff
@@ -46,7 +46,7 @@ This also assumes you have [ZooKeeper](http://zookeeper.apache.org/releases.html
 Before running the apps, you should do a `mvn clean install -DskipTests` in the Druid source in order to make sure directories are populated correctly.
 
 ## XML App Def
-You can configure application definitions in XML for import into IntelliJ. Below are a few examples. These should be placed in an XML file in [.idea/runConfigurations](.idea/runConfigurations) in the Druid source code.
+You can configure application definitions in XML for import into IntelliJ. Below are a few examples. These should be placed in an XML file in `.idea/runConfigurations` in the Druid source code.
```

Comment:
Same suggestion about retaining links 
```
[`.idea/runConfigurations`](/.idea/runConfigurations)
```

Reply:
@clintropolis 
Thank you!

If the suggestion above, how about changing the ./idea/ in `teamcity.md` file?
https://github.com/apache/druid/blame/master/dev/teamcity.md#L87

```diff
- If you did everything right, file `.idea/inspectionProfiles/Druid.xml` should be updated with a new
+ If you did everything right, file [`.idea/inspectionProfiles/Druid.xml`](/.idea/inspectionProfiles/Druid.xml) should be updated with a new
```

Reply:
sounds good :+1: